src/main/java/mobi/appcent/helium/api/ChallengesApi.java

```diff
@@ -0,0 +1,22 @@
+package mobi.appcent.helium.api;
+
+import mobi.appcent.helium.request.challenge.ListOfChallengesReceiptRequest;
+
+/**
+ * Created by erenalpaslan on 27.11.2022
+ */
+public class ChallengesApi extends BaseApi implements IChallengesApi{
+    @Override
+    String path() {
+        return null;
+    }
+
+    public static ChallengesApi getInstance() {
+        return new ChallengesApi();
```

Comment:
Is this method necessary? It is same calling the constructor right?

Reply:
Yes we can remove this method from all api classes